full_bubble_model.py

```diff
@@ -917,11 +965,11 @@ def print_data(cpar, data, print_it=True):
     text += f'''\nSimulation info:
     error_code ={data.error_code: .0f} (success = {data.success})
     elapsed_time ={data.elapsed_time: .2f} [s]
-    steps ={data.steps: .0f} [-]'''
+    nstep = {data.nstep};   saved_steps = {data.saved_steps};   nfev = {data.nfev};   njac = {data.njac};   nlu = {data.nlu}'''
```

Comment:
`data` now contains a bit more information of the solver, like nsteps, nfev, njac, nlu, nlu_jac... (see `OdeResult `in `scipy_ivp.py`)
Also t_last is saved and printed (under Final state:) to see where the solver failed.

```
Control parameters:
    ID = 1029,                                   # ID of control parameter (not used during calculation)
  # Initial conditions:
    R_E =  5.397959e-05,                         # bubble equilibrium radius [m]
    ratio =  1.00,                               # initial radius / equilibrium radius R_0/R_E [-]
    gases = [par.index['H2'], par.index['N2']],  # indexes of species in initial bubble (list of species indexes)
    fractions = [0.75, 0.25],                    # molar fractions of species in initial bubble (list of fractions for every gas)
  # Ambient parameters:
    P_amb =  101325.00,                          # ambient pressure [Pa]
    T_inf =  293.15,                             # ambient temperature [K]
  # Liquid parameters:
    alfa_M =  0.3500,                            # water accommodation coefficient [-]
    P_v =  2338.34,                              # vapour pressure [Pa]
    mu_L =  0.001018,                            # dynamic viscosity [Pa*s]
    rho_L =  998.20,                             # density [kg/m^3]
    c_L =  1483.00,                              # sound speed [m/s]
    surfactant =  1.00,                          # surfactant (surface tension modfier) [-]
  # Excitation parameters: (excitation_type = sin_impulse)
    p_A = -214285.71,                            # [Pa]
    freq =  20000.00,                            # [Hz]
    n =  1.00,                                   # [-]

Simulation info:
    error_code = 0 (success = True)
    elapsed_time = 8.03 [s]
    nstep = 12836;   saved_steps = 775;   nfev = 50298;   njac = 1692;   nlu = 1692 <------------------------------

Final state:
    R_final = 53.74 [um];   R_dot_final =-2.736321145541557e-11 [m/s];   T_final = 293.15 [K]
    n_NH3_final = 3.65e-13 [mol];   t_final = 1.000000e+00 [s] <------------------------------
    Final molar concentrations: [mol/cm^3]
        NH3   :  5.610779e-07;    H2    :  3.158170e-05;    H     :  4.025936e-20;    NH2   :  3.711937e-22;    
        NH    :  1.520492e-23;    N     :  5.156507e-12;    NNH   :  1.165822e-30;    N2H4  :  8.559234e-12;    
        N2H3  :  5.593610e-11;    N2H2  :  1.159118e-17;    H2NN  :  1.395565e-18;    N2    :  1.052720e-05;    
        
Results:
    collapse_time =  4.253892e-05 [s]
    T_max = 4891.06 [K]
    expansion work =  0.000000e+00 [J]
    dissipated acoustic energy =  5.673249e-06 [J]
    energy demand =  913.11 [MJ/kg of NH3]
```